types/cloudflare-turnstile/index.d.ts

```diff
@@ -64,16 +115,37 @@ declare namespace Turnstile {
         /**
          * Optional. A JavaScript callback that is invoked when a challenge expires.
          */
-        "expired-callback"?: VoidFunction;
+        "expired-callback"?: (token: string) => void;
+
+        /**
+         * Optional. A JavaScript callback that is invoked when an error occurs with the widget.
+         */
+        "error-callback"?: VoidFunction | undefined;
```

Comment:
`| undefined` is redundant because of `?`, here and below?

Reply:
It was done on all the previous types, I think it allows explicit undefined values, which can be helpful if the user passes a variable directly, that might be undefined